lms/djangoapps/learner_dashboard/views.py

```diff
@@ -139,3 +124,68 @@ def get(self, request, program_uuid):
             }
         }
         return Response(response_data, status=status.HTTP_200_OK)
+
+
+class ProgramLiveIframeView(ProgramDiscussionIframeView):
```

Comment:
I think we discussed to use permissions here and remove dependency on `ProgramDiscussionIframeView`?

```
    authentication_classes = (JwtAuthentication, BearerAuthentication, SessionAuthentication)
    permission_classes = (permissions.IsAuthenticated,)
```